src/app/modules/group/components/group-manager-list/group-manager-list.component.html

```diff
@@ -1,41 +1,47 @@
-<alg-section icon="fa fa-users" label="Managers of this group">
-  <alg-grid
-  [data]="managers"
-  sortMode="single"
-  [showGear]="false"
-  [scrollable]="true"
-  class="slanted-grid"
->
+<p-progressSpinner *ngIf="state === 'loading'"></p-progressSpinner>
 
-<ng-template #headerTemplate let-columns>
-  <tr style="border: none;">
-    <th style="min-width: 110px;">
-      <div class="slanted-header">
-        <div class="slanted-header-content">
-          Name
-        </div>
-      </div>
-    </th>
-    <th style="width: 160px; transform: none;">
-  </tr>
-</ng-template>
+<p *ngIf="state === 'error'">Error while loading the group managers</p>
 
-  <ng-template
-    #bodyTemplate
-    let-rowData
-    let-columns="columns"
-    let-rowIndex="rowIndex"
-  >
-    <tr [pSelectableRow]="rowData" [pSelectableRowIndex]="rowIndex">
-      <td
-        style="text-align: left;width: 160px;padding-left: 0.5rem;"
-        tooltipPosition="top"
-        tooltipStyleClass="tooltip-custom"
-      >
-        {{ rowData.name }}
-      </td>
-    </tr>
-  </ng-template>
-</alg-grid>
+<ng-container *ngIf="state === 'ready'">
```

Comment:
you could have put the `*ngIf="state === 'ready'"` on `alg-section ` without wrapping everything.